lib/LaTeXML/Engine/plain.pool.ltxml

```diff
@@ -159,8 +159,6 @@ RawTeX(<<'EoTeX');
   \mathchardef\@m=1000
   \mathchardef\@M=10000
   \mathchardef\@MM=20000
-  \countdef\m@ne=21\relax
-  \m@ne=-1
```

Comment:
Why do we lose `\m@ne` ? I have a very vague old memory of stumbling on an error with it a long time ago, as it turns out in #342 .

Reply:
Nope, we just lost the redefinition, which allocated it in the wrong slot, besides!